components/tools/OmeroPy/src/omero/plugins/admin.py

```diff
@@ -1646,65 +1687,11 @@ def reindex(self, args, config):
 
     @with_rw_config
     def ports(self, args, config):
-        self.check_access()
-        from omero.install.change_ports import change_ports
-        webserverkey = 'omero.web.application_server.port'
-        webserver_default_port = 4080
-        weblistkey = 'omero.web.server_list'
-        weblist_default_port = 4064
-        weblist_template = '[["localhost", %s, "omero"]]'
-
-        if not args.skipcheck:
-            if 0 == self.status(args, node_only=True):
-                self.ctx.die(
-                    100, "Can't change ports while the server is running!")
-
-            # Resetting return value.
-            self.ctx.rv = 0
-
-        if args.prefix:
-            for x in ("registry", "tcp", "ssl", "webserver"):
-                setattr(args, x, "%s%s" % (args.prefix, getattr(args, x)))
-        change_ports(
-            args.ssl, args.tcp, args.registry, args.revert, dir=self.ctx.dir)
-
-        # Use the same conditions as change_ports when modifying ports
-        if args.revert:
-            webserver_from = args.webserver
-            webserver_to = str(webserver_default_port)
-            weblist_from = weblist_template % args.ssl
-            weblist_to = weblist_template % str(weblist_default_port)
-        else:
-            webserver_from = str(webserver_default_port)
-            webserver_to = args.webserver
-            weblist_from = weblist_template % str(weblist_default_port)
-            weblist_to = weblist_template % args.ssl
-        try:
-            waport = config[webserverkey]
-        except KeyError:
-            waport = ''
-            webserver_from = ''
-        try:
-            weblist = config[weblistkey]
-        except KeyError:
-            weblist = ''
-            weblist_from = ''
-
-        if waport != webserver_from:
-            self.ctx.out('No match found for %s=%s in %s' % (
-                webserverkey, webserver_from, config.filename))
-        else:
-            config[webserverkey] = webserver_to
-            self.ctx.out('Converted: %s => %s %s in %s' % (
-                webserver_from, webserver_to, webserverkey, config.filename))
-
-        if weblist != weblist_from:
-            self.ctx.out('No match found for %s=%s in %s' % (
-                weblistkey, weblist_from, config.filename))
-        else:
-            config[weblistkey] = weblist_to
-            self.ctx.out('Converted: %s => %s %s in %s' % (
-                weblist_from, weblist_to, weblistkey, config.filename))
+        self.ctx.err(
+            "WARNING: the admin ports subcommand is deprecated. Changes will"
+            " be overwritten the next time the configuration files are"
+            " regenerated. Use the omero.ports.xxx configuration properties"
+            " instead.")
```

Comment:
At the moment `omero admin ports --help` returns the full help, no mention of deprecation/removal. So whatever is decided the help also needs to be changed/removed.